index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Lato Font Link Family/Style/Size -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <!-- Font Awesome Link -->
+    <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css" rel="stylesheet">
+    <link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
+    <!-- Free FontAwesome Link -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css" integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous">
+    <!-- Bootstrap Link -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <!-- Style Sheet Link -->
+    <link rel="stylesheet" href="style.css">
+    <title>The Way of Humanity - Flood Victims</title>
+</head>
+<body>
+    <section id="homepage">
+        <!-- Header Menu-1-2 -->
+        <header id="header-menu-1-2">
+            <!-- Toolbar header 1 -->
+            <div id="header-menu-1" class="">
+              <!-- Toolbar Name -->
+              <div id="spacer-menu-1" class="">
+                
+              </div>
+              <!-- Toolbar Menu -->
+              <nav class="">
+                <a href="#" class="header-icons"><i class=""></i></a>
+                <a href="#" class="header-icons"><i class=""></i></a>
+              </nav>
+              <nav class="" style="width: 20%; display: flex; flex-direction: row; justify-content: space-between;">
+                <a href="#english" style="color: #d3d3d3; font-size: 0.8rem;"><b>English</b></a>
+                <a href="#mypage" style="color: #d3d3d3; font-size: 0.8rem;"><b>My Page</b></a>
+                <a href="#logout" style="color: #d3d3d3; font-size: 0.8rem;"><b>Logout</b></a>
+              </nav>
```

Comment:
> * [ ]  Good job with the navbar 👍👍 However, as suggested by the reviewer [here](https://github.com/Abdulwhbhussain/Microverse-Capstone-M1/pull/2#discussion_r1173386509), please makes sure to include the social media icons as well, so that your design looks almost identical to the design guidelines ⚒ See the images below for more clarity 💡
> 
> Your design 🔽	Design guidelines 🔽
> ![image](https://user-images.githubusercontent.com/106119105/233692575-00f23082-1898-4a2f-8e1d-f04d3b3de193.png)	![image](https://user-images.githubusercontent.com/106119105/233692662-c9a3aa6f-2499-4e4e-b218-09c1a3bd9aef.png)

- As suggested by the code reviewer here, kindly include the social media icons. 

<img width="445" alt="image" src="https://user-images.githubusercontent.com/41163571/233743797-06e85db4-83e1-4c8a-bf41-bc98262610c7.png">